tendrl/commons/flows/create_cluster/utils.py

```diff
@@ -88,6 +88,12 @@ def install_python_gdeploy():
             "Failed to install python-gdeploy"
         )
 
+    # Write the python-gdeploy configuration file
+    config_data = "[python-gdeploy]\n" + \
+        "glusterfs_repo = https://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.9/"
```

Comment:
how does this work if i want to use a different glusterfs repo and not use the centos repo